### PR TITLE
Update keyboard shortcut in product page for preview

### PR DESCRIPTION
**Pull request by Quetzacoalt91**
Cherry pick of #7196 for 1.7.0.x branch

| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | BO: Update keyboard shortcut in product page for preview
| Type?         | bug fix
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | http://forge.prestashop.com/browse/BOOM-1755
| How to test?  | Go to the product page in the BO and test CTRL+E

### CODE REVIEW

File: src/PrestaShopBundle/Resources/views/Admin/Product/form.html.twig [hunk 1]
```diff
@@ -989,7 +989,7 @@
           class="btn btn-tertiary btn-submit preview"
           data-toggle="tooltip"
           id="product_form_preview_btn"
-          title="{{ 'See how your product sheet will look online: CTRL+V'|trans({}, 'Admin.Catalog.Help') }}"
+          title="{{ 'See how your product sheet will look online: CTRL+E'|trans({}, 'Admin.Catalog.Help') }}"
```
Comment (aleeks) [inline src/PrestaShopBundle/Resources/views/Admin/Product/form.html.twig]: You can't do this for a patch isn't ? @maximebiloe

Comment (xBorderie) [inline src/PrestaShopBundle/Resources/views/Admin/Product/form.html.twig]: sniwftr

...but I would ask @AlexEven first :)

**Comment by maximebiloe** [comment]
Yep, @aleeks is right.
As it's already merged in `develop`, I close this one.

**Comment by dave-ps** [comment]
http://www.howtogeek.com/114518/47-keyboard-shortcuts-that-work-in-all-web-browsers/

control - e is used already

Ctrl+K, Ctrl+E – Focus the browser’s built-in search box or focus the address bar if the browser doesn’t have a dedicated search box. (Ctrl+K doesn’t work in IE, Ctrl+E does.)

**Pull request closed without merging**